_pages/en/it-picture-medium-term.md

```diff
@@ -18,7 +18,7 @@ In the following sections, we're trying to take the same end state and create "v
 
 ## IITB-ESDC
 
-- IITB controls 20% of it's funding for Internal IT Projects 
+- IITB strictly controls 20% of it's funding for Internal IT Projects 
 - IITB is involved in business discussions and decisions (TODO: Are we now? What body should we be apart of, and who should be a part of it?)
 - 75% of employees are equipped with mobile devices allowing them to work from anywhere (TODO: Map trend and profiles on to metric )
```

Comment:
We're talking ESDC employees or IITB only?

Reply:
Well, I didn't add that in this pull request but I agree. That's why we have the TODO marker there. That needs to be researched and nuanced. 

We need to reference literature that allows for us to say that, and add profiles and nuance the statement